src/components/staking/PageValidator.vue

```diff
@@ -6,11 +6,16 @@
     :error="delegates.error"
     :data-empty="!validator"
     data-title="Validator"
+    :hide-header="true"
+    class="small"
   >
     <template v-if="validator" slot="managed-body">
-      <!-- we need the v-if as the template somehow is rendered in any case -->
-      <div class="page-profile__header page-profile__section">
-        <div class="row">
+      <div class="status-container">
+        <span :class="statusColor" class="validator-status">{{ status }}</span>
```

Comment:
No tooltips anymore? Don't we want to explain things?

Reply:
the tooltips were not very good at explaining things. yes, we want to explain things. we'll have to find a better way.